### DISCUSSION TITLE
OpenRoad Placement

### DISCUSSION THREAD

**Discussion by faisaladilquadery** [The-OpenROAD-Project/OpenROAD-flow-scripts#1638] [General] [answered]
I have noticed while working on a design in OpenRoad, I have improved setup timing violations by increasing utilization and placement density.
My question is that : 
Is openroad placement engine timing driven? If yes, to which extent? 
N.B  I have used switch global_placement -timing_driven but didnt notice significant improvement in setup violations which i am facing currently

**Comment by maliberty** [comment]
With -timing_driven it will periodically call the timing engine to adjust net weights (ie reduce wire length).  Beyond that general description a test case would be needed to see what's going on.

**Comment by faisaladilquadery** [comment]


**Reply by maliberty** [reply]
Increasing utilization/density will make the design more compact and the wires shorter leading to better timing.  However pushing it too far will result in flow failures as the design becomes to hard to optimize and route.

**Comment by maliberty** [comment, marked as answer]
Increasing utilization/density will make the design more compact and the wires shorter leading to better timing. However pushing it too far will result in flow failures as the design becomes to hard to optimize and route.

**Comment by maliberty** [comment]
You test case contained proprietary data that should not be made public so I deleted your comment.

**Comment by faisaladilquadery** [comment]
Thank you @maliberty  for the information
